staging/src/kubevirt.io/client-go/kubecli/version.go

```diff
@@ -54,7 +55,10 @@ func (v *ServerVersion) Get() (*version.Info, error) {
 
 	var group metav1.APIGroup
 	// First, find out which version to query
-	uri := ApiGroupName
+	re := regexp.MustCompile(`(.*)/apis.*`)
```

Comment:
Hm... I think this may not work as expected if there is already an `apis` element in the path. What about instead parsing the URL from the config? Like its done in:

https://github.com/kubevirt/kubevirt/blob/74781324b726a6d58799be1257a34f61e6749513/staging/src/kubevirt.io/client-go/kubecli/vmi.go#L135-L141